cmake/std/PullRequestLinuxCudaDriver.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -l
+
+if [ "${BSUB_CTEST_TIME_LIMIT}" == "" ] ; then
```

Comment:
@bartlettroscoe @ZUUL42 @william76 

> Is BSUB_CTEST_TIME_LIMIT a system defined environment variable or is this a new Jenkins parameter that's being introduced?
If this doesn't exist, can it break things?

I am not certain, but I think it is just an environment variable that can be defined by a user. 
@bartlettroscoe may be able to confirm or correct this.
If it is not defined, it is set to a default value, so it shouldn't break things. The variable is specific to this build as far as current PR tests go (because ride uses bsub).

Reply:
This was just an env var that allowed specialization in the individual driver scripts.  This came from the file:

* Trilinos/cmake/ctest/drivers/atdm/ride/local-driver.sh

which has:

```
#!/bin/bash -l

if [ "${BSUB_CTEST_TIME_LIMIT}" == "" ] ; then
  export BSUB_CTEST_TIME_LIMIT=12:00
fi

...

source $WORKSPACE/Trilinos/cmake/std/atdm/load-env.sh $JOB_NAME

set -x

bsub -x -Is -q $ATDM_CONFIG_QUEUE -n 16 -J $JOB_NAME -W $BSUB_CTEST_TIME_LIMIT \
  $WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/ctest-s-driver.sh
```

Any questions about this?